docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,521 @@
+You have 2 issues to resolve the problem:
+
+- Showing the HTML page in every resume event
+- No way to return to the main page of the Web application
+
+1. Avoid showing the HTML page in every resume event
+
+   The reason of this issue is that you used `loadURL` in the callback of resume event which occurs in every resuming of the Web application.
+   You have to move the code to invoke the API to somewhere else. In the list of addon events, you can find `lcPrelaunch` which occurs only once when a Web application runs. This event gives you a chance to do something only once just after you launch a Web application.
+
+   To handle the event, you have to add the handler again. You can use any name of it if it is not used for other event. Following shows the case if the name is `myPrelaunch`:may remember that you have removed the code lines related to `lcPrelaunch` when you tried to run the pure addon template. It is time to revive it and you need to move the `loadURL` from `resume()` to `myPrelaunch()`.
+
+   ```
+   activate() {
+       ...
+       this.evt.on('lcPrelaunch', this.myPrelaunch);
+       ...
+   } 
+   deactivate() {
+       ...
+       this.evt.off('lcPrelaunch', this.myPrelaunch);
+   }
+   myPrelaunch(winId, orig) {
+       addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');  
+   }
+   ...
+   ```
+
+   You can see that the `loadURL` has been moved from `resume()` to `myPrelaunch()`.
+
+2. Finding the way to return to the main page of the Web application
+
+   Now, we can control the frequent loading of the addon's HTML page. However, we are not able to return to the main page of the Web application.
+
+   How about if we can get the URL of the main page of the Web application ? We have the way !
+
+   In the parameter of `myPrelaunch()` above, you can find `orig`. That has the URL of the main page of the Web application. You need to keep & use it when you need. If you want to return to the main page when it goes to the suspend state, i.e. `lcSuspend` event. The implementation will be like:
+
+   ```
+   constructor(em) {
+       ...
+       this.org = null;
+   }
+   activate() {
+       ...
+       this.evt.on('lcSuspend', this.returnToMain);
+       ...
+   }
+   deactivate() {
+       ...
+       this.evt.off('lcSuspend', this.returnToMain);
+       ...
+   }
+   myPrelaunch(winId, orig) {
+       this.org = orig;
+       ...
+   }
+   resume(winId) {
+       ...
+   }
+   returnToMain(winId) {
+       addonAPI.window.loadURL(winId, this.org);
+   }
+   ```
+
+   You can test the implementation by the sequences as below :
+
+   1. In the Web application, press Home key to enter the suspend state
+   2. Long-press the Home key and select Basic UI to enter the resume state
+
+### Addition : Return to main page of Web application by back key
+
+In showing `hello.html` with previous solution, there will be no reaction when you press the Back Key.
+To resolve it, you have to change `hello.html` as following:
```

Comment:
Request you to please provide some more information so that I can suggest a rephrase.

Also, please change the second sentence to:

To resolve this issue, modify `hello.html` as shown in the following code:

Reply:
Here's my preference including more information:

When you press the **Back** key in the `hello.html` page, nothing happens. It is usually expected to return to a previous page from the current page after pressing the **Back** key.
To implement it, modify `hello.html` as shown in the following code:

Please verify